drivers/adc/CMakeLists.txt

```diff
@@ -16,10 +16,11 @@ zephyr_library_sources_ifdef(CONFIG_ADC_VF610		adc_vf610.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_SAM_AFEC	adc_sam_afec.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_NRFX_ADC	adc_nrfx_adc.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_NRFX_SAADC	adc_nrfx_saadc.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_SAM		adc_sam.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_SAM			adc_sam.c)
```

Comment:
Is your editor using TAB 4 instead 8? Check all files to use correct indentation.

Reply:
No, editor is set to TAB 4

Reply:
You should configure your IDE to use TAB 8 (zephyr default) and adjust if appropriated the indentation.